content/webapp/components/EventsByMonth/EventsByMonth.tsx

```diff
@@ -96,38 +66,25 @@ class EventsByMonth extends Component<Props, State> {
           </CssGridContainer>
         </Space>
 
-        {monthHeadings.map(month => (
+        {groups.map(g => (
           <div
-            key={month.id}
+            key={g.id}
             className={classNames({
               [cssGrid({ s: 12, m: 12, l: 12, xl: 12 })]: true,
             })}
             style={{
               display: !activeId
                 ? 'block'
-                : activeId === month.id
+                : activeId === g.id
                 ? 'block'
                 : 'none',
             }}
           >
-            <CssGridContainer>
-              <div className="css-grid">
-                <h2
-                  id={month.id}
-                  className={classNames({
-                    tabfocus: true,
-                    [cssGrid({ s: 12, m: 12, l: 12, xl: 12 })]: true,
-                  })}
-                >
-                  {month.id}
```

Comment:
This gets rid of the numeric date below the segmented control, because it's moderately annoying to reproduce in the new setup and I'm not sure it's actually useful. I can put it back if people want, but this code is all driven by month names now.

Before:

<img width="834" alt="Screenshot 2022-09-08 at 17 14 43" src="https://user-images.githubusercontent.com/301220/189173241-cc29d914-1ae3-4a06-a959-b8062fd9d576.png">

After:

<img width="838" alt="Screenshot 2022-09-08 at 17 14 47" src="https://user-images.githubusercontent.com/301220/189173244-44dfd954-b0ee-4112-a289-a09ad16d3b29.png">

Reply:
These were headings with IDs that are required as anchor links in the event that JS isn't available (the month names in the SegmentedControl link you to the appropriate part of the page). I'll put them back in (the pa11y dashboard tells us that there are links without named anchors on the page).

Reply:
#8439